docs/development/index.rst

```diff
@@ -10,6 +10,52 @@ Note that Astropy has very extensive developer documentation
 `here <http://astropy.readthedocs.org/en/latest/#developer-documentation>`__,
 this page should only mention Gammapy-specific things.
 
+.. _make_clean:
+
+How to clean up old files
+-------------------------
+
+Many projects have a ``Makefile`` to build and install the software and do all kinds of other tasks.
+In Astropy and Gammapy and most Python projects, there is no ``Makefile``, but the ``setup.py`` file
+and you're supposed to type ``python setup.py <cmd>`` and use ``--help`` and ``--help-commands`` to
+see all the available commands and options.
+
+There's one common task, cleaning up old generated files, that's not done via ``setup.py``.
+The equivalent of ``make clean`` is:
+
+.. code-block:: bash
+
+    $ rm -r build docs/_build docs/api htmlcov
+
```

Comment:
@mapazarr – Completely agree. There is one or several clean commands. I don't know what they do exactly or if they could be modified in Astropy or Gammapy to do exactly what we want. If you want you can look into this, but my guess is that it's hard to understand, especially for someone like you or me that's not a Python [setuptools](https://pythonhosted.org/setuptools/) expert.